inflammation/models.py

```diff
@@ -32,3 +32,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
```

Comment:
Function could use a clearer name for example standard_deviation.

Reply:
Resolved with commit 1670f1124d65ddad11b440678d4a192e5d29bcf8